database/seeds/AssessmentItemTableSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\AssessmentItem;
+
+class AssessmentItemTableSeeder extends Seeder
```

Comment:
It is overkill to have one seeder per table at this point.

I would just combine all of these in DatabaseSeeder.